src/server/pfs/server/driver.go

```diff
@@ -2417,7 +2417,11 @@ func (d *driver) getTreeForOpenCommit(pachClient *client.APIClient, file *pfs.Fi
 	return tree, nil
 }
 
-func (d *driver) getFile(pachClient *client.APIClient, file *pfs.File, offset int64, size int64) (r io.Reader, retErr error) {
+// getFile traverses the nodes of an arbitrary glob pattern
+// ultimately generating a stream of bytes (in the form of a reader)
+// of the object storage references containing the data for each node.
+// This byte stream m
```

Comment:
I am guessing there was more to comment here?

Reply:
If there was, I have since forgotten. Deleted.